cli/proc_state.rs

```diff
@@ -765,6 +779,20 @@ impl ProcState {
   }
 }
 
+pub fn import_map_from_value(
```

Comment:
This seems a little too low level for proc_state.rs and for re-use in the lsp for this to live here. Could we move this to cli/args? Maybe it could be done in the `to_import_map` method.

Reply:
It's also suspect that `import_map_from_text` is here and being re-used in the lsp.